app/frontend/components/shared/form/input-form-control.tsx

```diff
@@ -78,7 +78,8 @@ export const TextAreaFormControl = (props: IInputFormControlProps) => {
         {
           inputProps: { as: Textarea }, // Use Chakra's Textarea instead of Input
           validate: {
-            satisfiesLength: (str) => !props.required || (str?.length >= 1 && str?.length < 256) || "Input is invalid", // Use a default error message or use translation as needed
+            satisfiesLength: (str) =>
+              ((!props.required || str?.length >= 1) && str?.length < 512) || "Input is invalid", // Use a default error message or use translation as needed
```

Comment:
@bkmaibach I think the logic should be `(!props.required && !str) || (str?.length >= 1 && str?.length < 512) || t("ui.invalidInput")`. The same thing on the one above